# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/lighttpd/meson.patch.cross
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- lighttpd-1.4.59/meson_options.txt	2021-02-02 15:11:28.000000000 +0200
+++ meson_options.txt	2021-08-28 21:34:42.644256657 +0300
@@ -148,7 +148,11 @@
 	value: true,
 	description: 'with deflate-support for mod_deflate [default: on]',
 )
-
+option('with_zstd',
+	type: 'boolean',
+	value: false,
+	description: 'with zstd-support for mod_deflate [default:off]',
+	)
 option('build_extra_warnings',
 	type: 'boolean',
 	value: false,
